app-tasks/rf/src/rf/cli.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+"""Console script for Raster Foundry"""
+
+import logging
+
+import click
+
+from .commands import (
+    export,
+    find_aoi_projects,
+    ingest_scene,
+    process_upload,
+    update_aoi_project
+)
+from .utils.exception_reporting import wrap_rollbar
+
+logger = logging.getLogger('rf')
```

Comment:
Is this going to print to `stdout` properly?

Reply:
I believe it does by default (at least so in my testing).